standard/conversions.md

```diff
@@ -725,6 +737,7 @@ Specifically, an anonymous function `F` is compatible with a delegate type `D`
 
 > *Example*: The following examples illustrate these rules:
 >
+> <!-- Example: {template:"standalone-lib", name:"AnonymousFunctionsConv1", expectedErrors:["CS1593","CS1661","CS1678","CS8030","CS1688","CS1661","CS1676","CS1643","CS0126","CS0029","CS1662","CS0029","CS1662"]} -->
```

Comment:
If we find that lines are getting very long, I can look at how hard it would be to allow multi-line comments with metadata...